draft-ietf-rtcweb-jsep.xml

```diff
@@ -5680,6 +5582,167 @@ a=rtcp-fb:100 nack pli
+          have one MID.</t>
+
+          <t>Construct a table mapping incoming SSRC to "m=" line for
+          each "m=" line in this BUNDLE group and for each SSRC
+          configured for receiving in that "m=" line.</t>
+
+          <t>Construct a table mapping outgoing SSRC to "m=line" for
+          each "m=" line in this BUNDLE group and for each SSRC
+          configured for sending in that "m=" line.</t>
+
+          <t>Construct a table mapping payload type to "m=" line for
+          each "m=" line in the BUNDLE group and for each payload type
+          configured for receiving in that "m=" line.  If any payload
+          type is configured for receiving in more than one "m=" line
+          in the BUNDLE group, do not it include it in the table.</t>
+
+          <t>Note that for each of these tables, there can only be one
+          mapping for any given key (MID, SSRC, or PT).  In other
+          words, the tables are not multimaps.</t>
+        </list>
+      </t>
+
+      <t>As "m=" lines are added or removed from the BUNDLE groups, or
+      their configurations are changed, the tables above must also
+      be updated.</t>
+
+      <t>For each RTP packet received, the following steps MUST be
+      followed to route the packet to the correct "m=" section within
+      a BUNDLE group.  Note that the phrase "deliver a packet to the
+      'm=' line" means to further process the packet as would normally
+      happen with RTP/RTCP if it were received on a transport
+      associated with that "m=" line outside of a BUNDLE group (if the
+      "m=" line were not BUNDLEd) including dropping an RTP packet if
+      the packet's PT does not match any PT in the "m=" line.</t>
+
+      <t>
+        <list>
+          <t>If the packet has a MID and that MID is not in the table
+          mapping MID to "m=" line, drop the packet and stop.</t>
+
+          <t>If the packet has a MID and that MID is in the table
+          mapping MID to "m=" line, update the incoming SSRC mapping
+          table to include an entry that maps the packet's SSRC to the
+          "m=" line for that MID.</t>
+
+          <t>If the packet's SSRC is in the incoming SSRC mapping
+          table, route the packet to the associated "m=" line and
+          stop.</t>
+
+          <t>If the packet's payload type is in the payload type
+          table, update the the incoming SSRC mapping table to include
+          an entry that maps the packet's SSRC to the
+          "m=" line for that payload type. In addition, route the packet
+          to the associated "m=" line and stop.</t>
+
+          <t>Otherwise, drop the packet.</t>
+        </list>
+      </t>
+
+      <t>For each RTCP packet received (including each RTCP packet
+      that is part of a compound RTCP packet), the packet MUST be
+      routed to the appropriate handler for the SSRCs it contains
+      information about. Some examples of such handling are given
```

Comment:
One might indeed wonder that, particularly since RTCP handling is actually pretty tricky (and has encountered interop issues).  Personally I thought the text in JSEP-17 (though imperfect) was a step in the right direction.

As an example, who do we determine the "handler for the SSRCs it contains information about"?  The obvious way (for a receiver) is via the SSRC_Table.  But that implies that it is affected by things like SSRC latching (for both MID and PT).